src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/DSA/DSASignatureFormatter.cs

```diff
@@ -0,0 +1,76 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Security.Cryptography.Tests;
+using Xunit;
+
+namespace System.Security.Cryptography.Dsa.Tests
+{
+    public partial class DSASignatureFormatterTests : AsymmetricSignatureFormatterTests
+    {
+        [Fact]
+        public static void VerifySignature_SHA1()
+        {
+            using (DSA dsa = DSAFactory.Create())
+            {
+                var formatter = new DSASignatureFormatter(dsa);
+                var deformatter = new DSASignatureDeformatter(dsa);
+                VerifySignature(formatter, deformatter, SHA1.Create(), "SHA1");
```

Comment:
I'd like to see the hash algorithm object creations put in using statements.  (Either I never made that comment or it got lost, because I can't find it now)